src/components/views/BrickGameView/Game.ts

```diff
@@ -352,7 +359,7 @@ class Brick {
         this.app.context.fillStyle = 'rgb(0, 240, 0)'; //Green
         break;
       case 2:
-        this.app.context.fillStyle = 'rgb(255,140,0'; //Orange?
+        this.app.context.fillStyle = 'rgb(255, 140, 0)'; //Orange? Looks orange to me :)
```

Comment:
Thanks for confirming 😅 Not sure why there was a question mark there

```suggestion
        this.app.context.fillStyle = 'rgb(255, 140, 0)'; //Orange
```